package.json

```diff
@@ -13,6 +13,7 @@
     "lint:fix": "eslint ./src/** --ext .js,.jsx,.ts,.tsx --fix"
   },
   "dependencies": {
+    "railgun-mini-utils": "github:kassandraoftroy/railgun-mini-utils#060a769da8873a9798b6f3076935cc60cb18075c",
```

Comment:
this is the "library code" that I think should live in the "sdk" and not in the ambire-common repo. Wasn't sure where to put it so just doing this for the time being.